### ISSUE TITLE
Expand on "Some settings from Starship"

### ISSUE THREAD

**Issue by jvanderen1** [warpdotdev/Warp#2612]
### Describe the bug

I am trying to set up my shell to be as Warp-friendly as possible. However, I notice some of the icons look a bit small for me, but doesn't happen in iTerm2:

![Screenshot 2023-02-05 at 1 01 51 AM](https://user-images.githubusercontent.com/22926257/216808161-aea413d4-6819-498d-b9df-0eb2a33e4eac.png)

I found the [Some settings from Starship](https://docs.warp.dev/help/known-issues#some-settings-from-starship) section in the documentation, but there's no context or reasoning added to the suggested setting. In fact, I tried creating a [Starship configuration file](https://starship.rs/config/) for the first time, but hit an error:

```shell
[WARN] - (starship::config): Failed to load config value: invalid type: boolean `true`, expected struct CustomConfig
```

Please update documentation to be more clear.

### To Reproduce

https://docs.warp.dev/help/known-issues#some-settings-from-starship

### Operating System

MacOS

### Shell Version

fish, version 3.6.0

### Does this block you from using Warp daily?

No

### Warp Internal (ignore): linear-label:b8107fdf-ba31-488d-b103-d271c89cac3e

None

**Comment by dannyneira** [comment]
Hi @jvanderen1 Thanks for submitting, if I understand you correctly, you found the setting in our known issues doc and applied it to your starship and it didn't work? afaik, those settings shouldn't be in Starship for it to work with Warp, but if that isn't the case we can update the docs to reflect this.

**Comment by jvanderen1** [comment]
@dannyneira Correct! I was following the known issues doc trying to fix my icon issues. But it seems to be causing other issues.

**Comment by jvanderen1** [comment]
I've tried to change the font to 3 different fonts ("Hack (default)", "FiraCode Nerd Font Mono", "Hack Nerd Font Mono") and it still results in the same small icons. I'm not sure how to change this behavior.

<img width="1792" alt="Screenshot 2023-02-09 at 10 21 20 PM" src="https://user-images.githubusercontent.com/22926257/218007653-a14c11fb-533b-49c1-9f0c-398b36614259.png">

**Comment by injust** [comment]
> Also for the starship error, comment out or delete the `disabled=true` & `'' = 'https://starship.rs/config-schema.json' lines in your `~/.config/starship.toml`
> 
> ```toml
> # Get editor completions based on the config schema
> #'' = 'https://starship.rs/config-schema.json'
> 
> [custom]
> #disable=true
> ```

@dannyneira Sorry to necro an old issue, but this was the only relevant discussion about https://docs.warp.dev/appearance/prompt#starship-settings.

---

1. Are there more details on the known errors that are caused by these lines in `starship.toml`?
```toml
# Get editor completions based on the config schema
'' = 'https://starship.rs/config-schema.json'

# Disables the custom module
[custom]
disabled = false
```

I ask because it seems like the config schema does what it says: it allows your editor to provide completions. It doesn't actually change the behaviour of Starship, so I don't understand what problems it could cause with Warp.

---

2. It seems like the first part is outdated. From https://starship.rs/config/#configuration, it should be:

```toml
# Get editor completions based on the config schema
"$schema" = 'https://starship.rs/config-schema.json'
```
